pkg/api/handler/crud_handlers.go

```diff
@@ -71,45 +71,77 @@ func getPayloadCfg(c echo.Context) PayloadCfg {
 		ShowChildren: false,
 	}
 
-	var queryParams map[string][]string
+	var (
+		queryParams map[string][]string
+		bulkGet     bool = false
+	)
 	queryParams = c.QueryParams()
 	if strings.Index(c.Path(), ":id") < 0 {
+		// get all case
+		bulkGet = true
 		if details, ok := queryParams["detail"]; ok && len(details) == 1 && details[0] == "true" {
 			cfg.ShowDetails = true
 		} else {
 			cfg.ShowDetails = false
 		}
+	}
+
+	// get by ID case
+	if _, ok := queryParams["strict_fields"]; ok {
+		cfg.StrictFields = true
+	}
+	if fields, ok := queryParams["fields"]; ok {
+		if len(fields) > 0 {
+			cfg.Fields = strings.Split(fields[0], ",")
+			if len(cfg.Fields) > 0 {
+				modelInfo, _ := models.GetModelInfo(entity)
+				for _, refType := range modelInfo.RefTypes {
+					if utils.IndexOf(cfg.Fields, refType+"_refs") >= 0 {
+						cfg.ShowRefs = true
+					}
+				}
+				for _, backRefType := range modelInfo.BackRefTypes {
+					if utils.IndexOf(cfg.Fields, backRefType+"_refs") >= 0 {
+						cfg.ShowBackRefs = true
+					}
+				}
+				for _, childType := range modelInfo.ChildTypes {
+					if utils.IndexOf(cfg.Fields, utils.Pluralize(childType)) >= 0 {
+						cfg.ShowChildren = true
+					}
+				}
+			}
+		}
 	} else {
-		if _, ok := queryParams["strict_fields"]; ok {
-			cfg.StrictFields = true
+		if excludeRefs, ok := queryParams["exclude_refs"]; ok && len(excludeRefs) == 1 && excludeRefs[0] == "false" {
+			cfg.ShowRefs = true
```

Comment:
Currently, by default for all microservices using Cassandra (tssm, ems-central), exclude_refs is always false.

Reply:
fixed